packages/calcite-components/src/tests/commonTests.ts

```diff
@@ -1839,3 +1839,158 @@ export function openClose(componentTagOrHTML: TagOrHTML, options?: OpenCloseOpti
     });
   }
 }
+
+export async function themed(
+  componentTagOrHTML: TagOrHTML,
+  tokens: string[],
+  componentName?: string,
+): Promise<{ page: E2EPage; component: E2EElement; themedTokens: Record<string, string> }> {
+  let colorList: string[] = [
```

Comment:
Do we really need all these colors and the randomness or could we just set everything to `hotpink` since I don't think we'll ever be using a color like that.

Reply:
nope. updated.